zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -2169,7 +2169,12 @@ angular.module('zeppelinWebApp').controller('ParagraphCtrl', function($scope, $r
       var row = $scope.paragraph.result.msgTable[r];
       var dsvRow = '';
       for (var index in row) {
-        dsvRow += row[index].value + delimiter;
+        var stringValue =  (row[index].value).toString();
```

Comment:
it would need to handle other cases too. to quote [here](https://en.wikipedia.org/wiki/Comma-separated_values#General_functionality)
"CSV implementations permit them, often by requiring " (double quote) characters around values that contain reserved characters (such as commas, double quotes, or less commonly, newlines). Embedded double quote characters may then be represented by a pair of consecutive double quotes,[10] or by prefixing an escape character such as a backslash (for example in Sybase Central)."

For instance, when the stringValue has `"`